Octokit.Tests/Clients/UserEmailsClientTests.cs

```diff
@@ -1,14 +1,14 @@
-﻿using NSubstitute;
-using System;
+﻿using System;
 using System.Collections.Generic;
 using System.Threading.Tasks;
+using NSubstitute;
 using Xunit;
 
 namespace Octokit.Tests.Clients
 {
     public class UserEmailsClientTests
     {
-        public class TheGetAllMethod
+        public class TheGetAllMethods
```

Comment:
Can we revert this back to `TheGetAllMethod` rather than pluralising it.  Not because there aren't more than one method now, but more just for consistency with the rest of the code base!